### PR TITLE
Update menu.ejs

### LINKED ISSUE #29: Enable cookies for website
Users should be able to have a cart that does not get deleted upon a page refresh, or navigating to a new tab.  We will need cookies to allow for this to happen.

### PR DESCRIPTION
**Pull request by ganheather02**
added storing info in JSON

Closes #29.

### CODE REVIEW

**Review by NirmalAgnihotri — DISMISSED** [review]

File: app/routes/menu/index.js [hunk 1]
```diff
@@ -8,9 +8,19 @@ router.get('/', (req, res, next) => {
...
+// Post menu request, add to cart.
 router.post('/', (req, res) => {
-  // console.log(req.body);
+  try {
+    // Attempt to push new order into cart.
+    const { cart } = req.cookies;
+    cart.push(req.body);
+    res.cookie('cart', cart);
+
+    res.cookie('cart', req.cookies.cart.push(req.body));
```
Comment (declansullivan) [inline app/routes/menu/index.js]: Remove this old line, this is what was giving us that `2` issue in the past

File: app/views/menu.ejs [hunk 2]
```diff
@@ -247,8 +247,71 @@
         </div>
     </div>
 
+    
     </body>
 
+    
+
+    <script type="text/javascript">
```
Comment (declansullivan) [inline app/views/menu.ejs]: Remove this old code for cookies

**Review by declansullivan — CHANGES_REQUESTED** [review]
Remove old cookie code, as well as old line that gave us the list size error during the meeting

**Review by declansullivan — APPROVED** [review]
LGTM

**Review by dhrvsood — APPROVED** [review]

**Pull request merged**